### PR TITLE
2.0.0 experimental.53

### LINKED ISSUE #519: release 49 not working in react native
I get the following error `error: bundling failed: Error: Unable to resolve module `crypto` from `node_modules/ethers/utils/random-bytes.js`: crypto could not be found within the project.` that I know is related to the way functions are imported from ethers.js 

Version 48 is working fine

### PR DESCRIPTION
**Pull request by orenyodfat**
this is same pr as #528 though vs master-2 @jellegerbrandy 

here is the original description:
"This is to resolve #519 , by changing back some BigNumber types to BN types.

Apparently the problem occurs is caused by importing BigNumber from ethers/utils, it also imports the (unrelated) crypto library, which is not available in the react native environment.

This is a quick pragmatic fix which addresses only the files that we are using in common, and is just meant to unblock the situation there. We should probably use a single type for large numbers. This issue addresses that: #529"

### CODE REVIEW

**Comment by orenyodfat** [comment]
see comments https://github.com/daostack/arc.js/pull/528#issuecomment-687717780

**Comment by orenyodfat** [comment]
@jellegerbrandy please update version to rc.55 (on top of 54).

File: src/plugins/join/plugin.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import BN from 'bn.js'
-import { BigNumber } from 'ethers/utils'
```
Comment (roienatan) [inline src/plugins/join/plugin.ts:2]: @jellegerbrandy Are you sure the error is thrown from this import? What about the other imports of BigNumber?

**Comment by orenyodfat** [comment]
seems like ethers.js use its own BigNumber .
is use BN.js internally.
https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--BigNumber--notes--why-not-bignumber-js-bn-js-bigdecimal-etc

**Comment by dkent600** [comment]
@orenyodfat @roienatan

@jellegerbrandy answered these questions in the description of the PR.

**Comment by roienatan** [comment]
I thought about a possible solution of importing BigNumber in a separate file and exporting it to the necessary modules. I want to try and see if I get the react native error now. How can I reproduce this? @jellegerbrandy

**Comment by orenyodfat** [comment]
see #538

**Pull request closed without merging**